client/src/modules/payroll/rubrics/rubrics.js

```diff
@@ -0,0 +1,110 @@
+      { field : 'is_discount', displayName : '', cellTemplate : '/modules/payroll/rubrics/templates/discount.tmpl.html', headerCellFilter : 'translate' },    
+      { field : 'value', displayName : 'FORM.LABELS.VALUE', headerCellFilter : 'translate' },  
+      { field : 'is_percent', displayName : 'FORM.LABELS.PERCENTAGE', cellTemplate : '/modules/payroll/rubrics/templates/percent.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'is_social_care', displayName : 'FORM.LABELS.IS_SOCIAL_CARE', cellTemplate : '/modules/payroll/rubrics/templates/social.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'is_tax', displayName : 'FORM.LABELS.TAX', cellTemplate : '/modules/payroll/rubrics/templates/tax.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'is_ipr', displayName : 'FORM.LABELS.IS_IPR', cellTemplate : '/modules/payroll/rubrics/templates/ipr.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'four_account_id', displayName : 'FORM.LABELS.FOUR_ACCOUNT', cellTemplate : '/modules/payroll/rubrics/templates/four.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'six_account_id', displayName : 'FORM.LABELS.SIX_ACCOUNT', cellTemplate : '/modules/payroll/rubrics/templates/six.tmpl.html', headerCellFilter : 'translate' },
+      { field : 'action',
+        width : 80,
+        displayName : '',
+        cellTemplate : '/modules/payroll/rubrics/templates/action.tmpl.html',
+        enableSorting : false,
+        enableFiltering : false,
+      },
+    ];
+
+  // options for the UI grid
+  vm.gridOptions = {
+    appScopeProvider  : vm,
+    enableColumnMenus : false,
+    fastWatch         : true,
+    flatEntityAccess  : true,
+    enableSorting     : true,
+    onRegisterApi     : onRegisterApiFn,
+    columnDefs : gridColumn,
+  };
+
+  function onRegisterApiFn(gridApi) {
+    vm.gridApi = gridApi;
+  }
+
+  function toggleFilter() {
+    vm.filterEnabled = !vm.filterEnabled;
+    vm.gridOptions.enableFiltering = vm.filterEnabled;
+    vm.gridApi.core.notifyDataChange(uiGridConstants.dataChange.COLUMN);
+  }
+
+  function loadRubrics() {
+    vm.loading = true;
+
+    Rubrics.read(null, { detailed : 1 })
+    .then(function (data) {
+      vm.gridOptions.data = data;
+    })
+    .catch(Notify.handleError)
+    .finally(function () {
+      vm.loading = false;
+    });
+  }
+
+  // switch to delete warning mode
+  function deleteRubric(title) {
+    ModalService.confirm('FORM.DIALOGS.CONFIRM_DELETE')
+    .then(function (bool) {
+      if (!bool) { return; }
+
+      Rubrics.delete(title.id)
+      .then(function () {
+        Notify.success('FORM.INFO.DELETE_SUCCESS');
+        loadRubrics();
+      })
+      .catch(Notify.handleError);
+    });
+  }
+
+  // update an existing Rubric
+  function editRubric(title) {
+    $state.go('rubrics.edit', { id : title.id });
+  }
+
+  // create a new Rubric
+  function createRubric() {
+    $state.go('rubrics.create');
```

Comment:
These state links should just be directives using `ui-sref`.  It will be easier to migrate to future versions of angular that way, I think.